### DISCUSSION TITLE
Reembolsos de ingressos PyNE 2024

### DISCUSSION THREAD

**Discussion by itepifanio** [apyb/comunidade#133] [Geral]
Sabemos que por lei o reembolso de ingressos é de apenas 7 dias após a compra, entretanto existem casos e casos, como por exemplo a edição de 2024 teve uma caravana de estudantes da UFPI que não conseguiu ir ao evento porque a universidade não liberou o ônibus como tinha feito no ano de 2023.

O evento de 2024 tem um pouco de caixa para reembolsar esses estudantes, alguns deles inclusive já foram reembolsados, mas conversando com o time da edição 2025 eles sugeriram reavaliar essa estratégia para garantir a sustentação do evento no ano seguinte. Como o evento é da comunidade acredito que essa decisão pode ser tomada de forma coletiva, então abro aqui a discussão.

Eu particularmente gostaria de garantir o reembolso desses estudantes, mas caso não seja feito sugiro a edição 2025 de gerar um cupom de desconto para esses participantes (que são inclusive da cidade que hospedará o evento, Teresina).

**Comment by cecivieira** [comment]
@itepifanio eu não vejo problema em reembolsar esses estudantes que tiveram o ônibus negado pela universidade.
Caso o evento não tivesse caixa, então ficaria um pouco difícil. Mas não é esse o caso.

**Comment by anapaulamendes** [comment]
Se temos caixa e a apyb está de acordo, vamos seguir com os reembolsos. Nosso receio era que estivesse fora da regra dos 7 dias para reembolsar e pudesse ter algum problema e de certa forma também esse caixa ajudar na edição atual. Mas estamos de acordo.

**Comment by cecivieira** [comment]
@anapaulamendes os dinehiros das edições não passam para a seguinte. Sugiro leitura do Estatuto da APyB e Guia de Organização de Eventos para entender o fluxo.
